rules-reviewed/eap7/eap4and5/jboss-eap4and5to6and7-xml.windup.xml

```diff
@@ -49,13 +49,12 @@
                 <hint issue-display-mode="all" title="Replace BarrierController service" effort="1" category-id="mandatory">
                     <message>
                         <![CDATA[
-                        `BarrierController` service is no longer available in JBoss EAP 6+.  
-                         In JBoss EAP 4.x the JBoss 'kernel' layer was an extension of JMX which is why everything was an MBean, though most of the MBean functionality still worked in JBoss EAP 5.  
-                         JBoss EAP 6 is no longer based on Mbeans, and the Mbeans which are seen in jconsole are actually just facades over the JBoss management API, so declaring Mbean dependencies on any other Mbeans that appear to be JBoss Mbeans does not work since they are not real Mbeans.  
```

Comment:
is this double space removal on purpose ? we put it to create a new line after the text.